lib/gst/plugin/gstclappersink.c

```diff
@@ -889,6 +967,8 @@ gst_clapper_sink_class_init (GstClapperSinkClass *klass)
 
   sink_pad_templ = gst_clapper_importer_loader_make_sink_pad_template ();
   gst_element_class_add_pad_template (gstelement_class, sink_pad_templ);
+
+  gst_type_mark_as_plugin_api (GST_TYPE_CLAPPER_SINK_ROTATION, 0);
```

Comment:
If we change to `GST_TYPE_VIDEO_ORIENTATION_METHOD` this won't be needed anymore.